components/fastly_backends/main.tf

```diff
@@ -32,8 +32,8 @@ locals {
     # when crossing domains, and nothing at all when going to HTTP.
     "Referrer-Policy" = "\"strict-origin-when-cross-origin\""
 
-    # Enforce HTTPS for all traffic (TODO: https://www.pivotaltracker.com/story/show/174911098)
-    "Strict-Transport-Security" = "\"max-age=300; includeSubDomains\"" # 5 minutes.
+    # Enforce HTTPS for all traffic (TODO: https://www.pivotaltracker.com/story/show/175288875)
+    "Strict-Transport-Security" = "\"max-age=604800; includeSubDomains\"" # 1 week.
```

Comment:
Is the thought on this bump that if we were gonna see issues, we would have seen them with 5 minutes? So it's okay to jump straight to a week? What kind of errors would we have seen if there were issues?

Reply:
Yeah, I went back and forth on whether to try something like a day or 3 days... this is just following Google's [deployment recommendations](https://hstspreload.org/#deployment-recommendations) from [hstspreload.org](https://hstspreload.org/).

If this didn't work, we'd be seeing Zendesk or staff reports of errors (like a "cannot connect" Chrome tab or something) where users couldn't load a `*.dosomething.org` subdomain that didn't support HTTPS. I don't know any off the top of my head, so I _think_ this unlikely though!

Reply:
@DFurnes ah okay so 5 minutes to 1 week is standard practice then! I'm okay with that, just was hesitant since we can't undo it

Reply:
Yeah, same! I think this should hopefully be pretty low-risk, and we'll wait a while before we bump it to a month. 🤞